client/src/components/Overview/index.jsx

```diff
@@ -42,7 +43,7 @@ class Overview extends React.Component {
             salePrice={salePrice}
           />
         </div>
-        <div className={styles.styleSelector}>Style Selector</div>
+        <div className={styles.styleSelector}><StyleSelector allStyles={allStyles} /></div>
```

Comment:
I have some questions regarding this and your import statement. Will reach out. (: